playbooks/deploy-external-ceph.yml

```diff
@@ -0,0 +1,253 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Pre-requisites for deploying external ceph
+  hosts: localhost
+  vars:
+    cifmw_ceph_dir: "{{ ansible_user_dir }}/ci-framework-data/manifests/ceph"
+    cifmw_ceph_node_num: "{{ cifmw_ceph_node_num | default(3) }}"
```

Comment:
Why not base the amount of ceph nodes in the size of a group called, ie, ceph, that it's part of the inventory used to deploy the external ceph?
Anyway, if this use-case is done with a DT I'm not sure why we need such of a playbook like this one.